front-end/src/components/Login/index.js

```diff
@@ -51,6 +51,10 @@ class Login extends Component {
             <label htmlFor="password">Пароль</label>
             <input type="password" name="password" id="password" placeholder="введите пароль (1234)"
               value={this.state.password} onChange={this.handleChange} required/>
+            <div className="remember">
```

Comment:
class name must begin with block name

Reply:
I've tried to do like BEM but the checkbox, didn't work properly, that way((

Reply:
+